google-cloud-bigquery/OVERVIEW.md

```diff
@@ -155,20 +155,21 @@ more complex types such as `ARRAY` and `STRUCT`.
 
 The BigQuery data types are converted to and from Ruby types as follows:
 
-| BigQuery    | Ruby           | Notes  |
-|-------------|----------------|---|
-| `BOOL`      | `true`/`false` | |
-| `INT64`     | `Integer`      | |
-| `FLOAT64`   | `Float`        | |
-| `NUMERIC`   | `BigDecimal`   | Will be rounded to 9 decimal places |
-| `STRING`    | `String`       | |
-| `DATETIME`  | `DateTime`     | `DATETIME` does not support time zone. |
-| `DATE`      | `Date`         | |
-| `TIMESTAMP` | `Time`         | |
-| `TIME`      | `Google::Cloud::BigQuery::Time` | |
-| `BYTES`     | `File`, `IO`, `StringIO`, or similar | |
-| `ARRAY`     | `Array` | Nested arrays and `nil` values are not supported. |
-| `STRUCT`    | `Hash`         | Hash keys may be strings or symbols. |
+| BigQuery     | Ruby                                 | Notes                                                        |
+|------------- |--------------------------------------|--------------------------------------------------------------|
+| `BOOL`       | `true`/`false`                       |                                                              |
+| `INT64`      | `Integer`                            |                                                              |
+| `FLOAT64`    | `Float`                              |                                                              |
+| `NUMERIC`    | `BigDecimal`                         | `BigDecimal` values will be rounded to scale 9.              |
+| `BIGNUMERIC` | converted to `BigDecimal`            | Pass data as `String` and map query param values in `types`. |
+| `STRING`     | `String`                             |                                                              |
+| `DATETIME`   | `DateTime`                           | `DATETIME` does not support time zone.                       |
+| `DATE`       | `Date`                               |                                                              |
```

Comment:
> https://github.com/googleapis/google-cloud-ruby/blob/7373fa0c45eacd2e631d9141e022e937425e9f73/google-cloud-bigtable/acceptance/bigtable/project_test.rb#L59-L81

Will do.

> I'm also noticing that we don't appear to have GEOGRAPHY type support in this library, which has been present in the service since circa 2018ish? Unclear what happened here, but we'll want to plumb this in as well (though not as part of this PR).

I seem to remember Mike asking about this and being told "not yet". Apparently we were too early to ask, then never asked again! Do you mind creating an issue for it? Would love to add it.

Reply:
We have this detection of the `GEOGRAPHY ` type:

https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-bigquery/lib/google/cloud/bigquery/standard_sql.rb#L331-L340

Reply:
Added https://github.com/googleapis/google-cloud-ruby/issues/11124 for the GEOGRAPHY support

Reply:
> If you're calling out TZ support, consider noting that all the civil types share this: DATETIME, DATE, TIME

After looking at this for a bit, I think it's best to only call out the lack of timezone support in `DATETIME` vs `DateTime`, since that's the only case in which the Ruby type **has** timezone support but the BQ type does not:

```
irb(main):003:0> dt = DateTime.now
=> #<DateTime: 2021-04-07T16:22:36-06:00 ((2459312j,80556s,557997000n),-21600s,2299161j)>
irb(main):004:0> dt.strftime "%Y-%m-%d %H:%M:%S.%6N"
=> "2021-04-07 16:22:36.557997"
irb(main):005:0> # Lost the time zone, ouch!
```

The Ruby types `Date` and `Google::Cloud::BigQuery::Time` do not have timezone support, so I think it only creates noise in the table to mention that the corresponding BQ types are similar.